pkg/apiserver/rest/dualwriter_mode2.go

```diff
@@ -42,12 +43,14 @@ func (d *DualWriterMode2) Create(ctx context.Context, obj runtime.Object, create
 	if err != nil {
 		return created, err
 	}
+
+	// create method expects an empty resource version
 	accessor.SetResourceVersion("")
 	accessor.SetUID("")
 
 	rsp, err := d.Storage.Create(ctx, c, createValidation, options)
 	if err != nil {
-		klog.Error("unable to create object in duplicate storage", "error", err, "mode", Mode2)
+		klog.FromContext(ctx).Error(err, "unable to create object in unified storage", "mode", Mode2)
```

Comment:
nit: from what I have seen the convention in this package and the related code has been to refer to unified storage as Storage.

Because modes are set using iota, I think Mode2 prints as `1`. 

```suggestion
		klog.FromContext(ctx).Error(err, "unable to create object in Storage", "mode", 2)
```